### ISSUE TITLE
Will it release

### ISSUE THREAD

**Issue by ralseilii** [CleanroomMC/Create#2]
I see this has no update for a year I wonder will create on 1.12.2 release one day,or is it abandoned?please

**Comment by IntegerLimit** [comment]
No, it has never received any updates, any commits, the only commit is just a default empty mod.

**Comment by Rongmario** [comment]
What do you mean? There has been numerous updates, you just can't see them that's all.

**Comment by KorewaLidesu** [comment]
> What do you mean? There has been numerous updates, you just can't see them that's all.

fr fr

**Comment by kappa-maintainer** [comment]
As far as I know, it is more an idea than a plan.
Working on our Forge fork already used all our manpower, and backporting Create is way larger than this
